src/features/rewards/tableContribute/style.ts

```diff
@@ -19,7 +19,7 @@ export const StyledRemove = styled<{}, 'button'>('button')`
   cursor: pointer;
   width: 21px;
   height: 21px;
-  color: #DFDFE8;
+  color: ${p => p.theme.color.disabled};
```

Comment:
Color change updated per @jenn-rhim (grey300)

Reply:
@ryanml is the semantic use of 'disabled' correct here? I'm not saying it isn't, just want to make sure that we want this tied to whatever color `disabled` is ever. Multiple entries in `theme.color` can point to grey300

Reply:
You're right, not quite semantic, will make second entry in `brave-defaults` for grey300